CMakeLists.txt

```diff
@@ -15,13 +15,6 @@ set(NVFUSER_THIRD_PARTY_DIR "${NVFUSER_ROOT}/third_party")
 option(NVFUSER_STANDALONE_BUILD_WITH_UCC "" OFF)
 option(NVFUSER_BUILD_WITH_ASAN "Build nvFuser with asan" OFF)
 
-include(CMakeDependentOption)
-cmake_dependent_option(NVFUSER_DISTRIBUTED "" ON "USE_DISTRIBUTED" OFF)
```

Comment:
don't we need to assert that `USE_DISTRIBUTED` in `ON`?

Reply:
Great point -- we should make the error show up early in config time not in build time.

Reply:
I added the assertion in setup.py. PTAL